aws-lambda/container/wsgi.py

```diff
@@ -0,0 +1,9 @@
+import os
```

Comment:
The CI tests run flake8 and this file throws flake8 error. Does the following work?
```python
import os

from pygeoapi.flask_app import APP

os.environ['PYGEOAPI_CONFIG'] = 'pygeoapi-test-config.yml'
os.environ['PYGEOAPI_OPENAPI'] = 'pygeoapi-test-openapi.yml'

if __name__ == "__main__":
    APP.run()
```